index.html

```diff
@@ -163,20 +169,41 @@ <h3>Skills</h3>
                             <li><a href="#">Terminal</a></li>
                         </ul>
                 </div>
-                
-
             </section>
 
         </section>
 
-
-
-
+        <section class="contact">
+            <p class="name">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</p>
+            <div class="div-contact">
+                <form class="contact-form" action="https://formspree.io/f/xayknejl" method="post">
+                    <input type="text" maxlength="30" placeholder="Full Name" required>
+                    <input type="email" placeholder="Email address" required>
```

Comment:
Kindly, the name and email fields must contain the NAME attribute for the form submission to work correctly.